RESTserver/src/main/java/com/softserve/if072/restservice/repository/ImageRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.softserve.if072.restservice.repository;
+
+import com.softserve.if072.common.model.Image;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+/**
+ * Created by Vitaliy Malisevych
+ */
+public interface ImageRepository extends JpaRepository<Image, Integer> {
+
+    List<Image> findAll();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)